### DISCUSSION TITLE
use directives with cli

### DISCUSSION THREAD

**Discussion by erwinheldy** [windicss/windicss#154] [Q&A]
how to compile directives using windicss cli ?
since windicss cli only accepts html files, how i put my components ?

**Comment by oliverandrich** [comment]
@erwinheldy Did you find out how to use directives with the CLI? I want to define some "components" in a CSS file using @apply, and then compile this file along with the classes extracted from the HTML into a single file.

**Reply by erwinheldy** [reply]
No, I didn't find out yet
